std/range/package.d

```diff
@@ -7172,25 +7172,25 @@ private struct OnlyResult(T, size_t arity)
 
     T front() @property
     {
-        assert(!empty);
+        assert(!empty, "Attempting to fetch the front of an empty only");
```

Comment:
yeah, this might be good for better uniformity across error messages from the different ranges.